Automattic/jetpack
# Update Creator product constants to be a product, not a plan.

Initially, the new Jetpack Creator product was setup in WPCOM to be a "Plan"/bundle, so in also in Jetpack plugin we added Creator as a Plan in https://github.com/Automattic/jetpack/pull/33893.  However, shortly after, we changed Jetpack Creator to be an individual add-on product, instead of a plan.
Therefore this PR makes the necessary updates in the Jetpack plugin to reflect Jetpack Creator as a "product" and not a "plan".  (See before and after images).

**BEFORE:**

<img width="1089" alt="279964067-b7bc7b0d-474a-4f9f-9b64-d776c87b6d70 (1)" src="https://github.com/Automattic/jetpack/assets/11078128/9aa9f2a6-8b16-4114-8b63-b30f535f514b">
.

**AFTER:**

![Markup 2023-11-07 at 14 52 04](https://github.com/Automattic/jetpack/assets/11078128/8a55c6c4-528e-4113-bb33-a41cb176a487)





## Proposed changes:

- Added/updated constants to reflect Jetpack Creator as a "product", instead of a "Plan".

### Other information:


## Jetpack product discussion


Internal reference: https://github.com/Automattic/jetpack-marketing/issues/316

## Does this pull request change what data or activity we track or use?


No

## Testing instructions:






- Apply the branch via Jetpack Beta on Jurassic Ninja site and connect Jetpack.
- Purchase Jetpack Creator for the site. (You can use A18N credits)
    - https://wordpress.com/checkout/:SITE/jetpack_creator_yearly
- Go to Jetpack > Dashboard > My Plan
- You should see the "Creator" product under the heading, "My products".
